lib/src/room.dart

```diff
@@ -1877,14 +1881,17 @@ class Room {
     return powerLevelMap.tryGet('users_default') ?? 0;
   }
 
-  /// The default level required to send message events. Can be overridden by the events key.
-  bool get canSendDefaultMessages =>
-      (getState(EventTypes.RoomPowerLevels)
-                  ?.content
-                  .tryGet<int>('events_default') ??
-              0) <=
-          ownPowerLevel &&
-      (!encrypted || client.encryptionEnabled);
+  /// The default level required to send message events. This checks if the
+  /// user is capable of sending `m.room.message` events.
+  /// Please be aware that this also returns false
+  /// if the room is encrypted but the client is not able to use encryption.
+  /// If you do not want this check or want to check other events like
+  /// `m.sticker` use `canSendEvent('<event-type>')`.
+  bool get canSendDefaultMessages {
+    if (encrypted && !client.encryptionEnabled) return false;
+
+    return canSendEvent(encrypted ? EventTypes.Encrypted : EventTypes.Message);
```

Comment:
Potential minor version change? This changes the previous behavior of sending events_default to the above events. (although afaik those are never set in powerlevels and canSendEvent should default to events_default later anyway)

Reply:
Will release a new major version as the next anyway :)